tiledb/sm/query/readers/reader_base.cc

```diff
@@ -1015,9 +1016,48 @@ Status ReaderBase::unfilter_tile_chunk_range(
     // Unfilter 't' for fixed-sized tiles, otherwise unfilter both 't' and
     // 't_var' for var-sized tiles.
     if (!var_size) {
-      if (!nullable)
-        RETURN_NOT_OK(unfilter_tile_chunk_range(
-            num_range_threads, range_thread_idx, name, t, tile_chunk_data));
+      if (!nullable) {
```

Comment:
Do we want to bitsort filter to work with Hilbert as well? This will not work with Hilbert.

Reply:
@ihnorton ?

Reply:
Ideally yes, but let's discuss level of effort to get it supported here.

Reply:
@ihnorton Since we want to squeeze it in 2.13, I think we could just return an error for now when the user tries to create a schema with bitsort and hilbert and focus on making the code for global order clean.

Reply:
like layout hilbert? tile or cell?

Reply:
Hilbert is a cell layout.